docs/source/install.rst

```diff
@@ -1,88 +1,50 @@
-.. _install:
+.. _Install:
 
-Installation
-************
-Instructions for downloading and installing Fab.
 
-Latest release
-==============
-You can install the latest release of Fab from PyPI::
-
-    pip install sci-fab
-
-Cutting Edge
-============
-If you want the newest features, you can download the repo from github::
-
-    git clone https://github.com/metomi/fab.git
-
-Then install the folder::
-
-    pip install ./fab
-
-
-Configuration
-=============
-
-You can optionally tell Fab where it's workspace should live.
-This can be useful on systems where your home space is on a slower network drive::
+Installing Fab
+**************
+You can install the latest release of `Fab from PyPI <https://pypi.org/project/sci-fab/>`_:
 
-    export FAB_WORKSPACE=/tmp/persistent/fab_workspace
+.. code-block:: console
 
-If you don't do this, Fab will create a project workspace underneath ``~/fab-workspace``.
+    $ pip install sci-fab
 
+The minimum Python dependencies (e.g. `fparser <https://github.com/stfc/fparser>`_)
+will also be installed automatically.
 
-Run Environment
-===============
-You must have an environment with the compilers and linkers you need to build your project.
-
-Conda
------
-You can create a conda environment for running fab from a yaml file in the repo::
-
-    git clone https://github.com/metomi/fab.git
-    conda env create -f fab/dev_envs/dev_env.yml
-
-Activate the new environment::
-
-    conda activate sci-fab
-
-Install fab::
+.. note::
 
-    pip install ./fab
+    When installing from PyPI, please be sure to install **sci-fab**, not *fab*.
+    There is already a package on PyPI called *fab*, which installs something else entirely.
 
 
+Extra features
+==============
+You can install some extra Python packages to enable more features.
+This will install `matplotlib <https://matplotlib.org/>`_ for producing metrics graphs after a run,
+`psyclone <https://github.com/stfc/PSyclone>`_ for building LFRic, and more.
 
-.. note::
-    *Internal Met Office Users*
+.. code-block:: console
 
-    **After** you've created your Conda environment with Fab installed,
-    you may need to activate modules in a new terminal to gain access to command line tools. E.g::
+    $ pip install sci-fab[features]
 
-        module use /data/users/lfric/modules/modulefiles.rhel7
-        module load environment/lfric/gnu
-        conda activate sci-fab
 
+Configuration
+=============
 
+.. _Configure Fab Workspace:
 
+Fab workspace
+-------------
 
-Developers
-==========
+You can optionally tell Fab where it's workspace should live.
+This can be useful on systems where your home space is on a slower drive::
```

Comment:
Or just when you like your build to be next to your source. It's a flexible option.